version.gradle

```diff
@@ -25,7 +25,7 @@
  * as we want to manage the versions in a single source.
  */
 
-final def spineVersion = '1.5.2'
+final def spineVersion = '1.5.3'
```

Comment:
There are many breaking changes in this version. I suggest we make a new minor update — `1.6.0`. As always, we can first publish this as `1.5.3` and check that it works and then advance to `1.6.0`.

Reply:
We will surely have 1.6.0 once other breaking changes are ready. @armiol is working on the other set of such, which is going to be way bigger. Changes in this PR are not that big, comparing with new storages API coming soon.

We surely do not abide to semantic versioning. But assuming that there are still not so many users of the framework, we'd better update quicker and easier rather than follow an artificial ballet with proper deprecation cycles.